recipe/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-sh Configure -de -Dprefix=$PREFIX -Duserelocatableinc
+sh Configure -Dusethreads -Duserelocatableinc -Dprefix=$PREFIX -de
```

Comment:
Also please see PR ( https://github.com/conda-forge/perl-feedstock/pull/5 ) where I was trying to do the same thing.